src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthConfiguration.java

```diff
@@ -56,4 +62,14 @@ public boolean autoDiscoverRealms()
     {
         return AUTO_DISCOVER_REALMS.getAsBoolean(this);
     }
+
+    public String getChallengeDeltaClaimNamespace()
+    {
+        return CHALLENGE_DELTA_CLAIM_NAMESPACE.get(this);
+    }
+
+    public String getChallengeDeltaClaimName()
+    {
+        return CHALLENGE_DELTA_CLAIM_NAME.get(this);
+    }
```

Comment:
Configuration accessor method names should also derive algorithmically from property names.